include/diploma.h

```diff
@@ -0,0 +1,6 @@
+#ifndef GUARD_DIPLOMA_H
+#define GUARD_DIPLOMA_H
+
+void CB2_ShowDiploma(void);
```

Comment:
Why is a header needed? It's only one function, called from one file.

Reply:
Isn't that enough reason?